Directory.Build.props

```diff
@@ -1,17 +1,38 @@
 <Project>
 
-  <Import Project="FSharpBuild.Directory.Build.props" Condition = " '$(FSharpTestCompilerVersion)' == '' "/>
-  <Import Project="FSharpTests.Directory.Build.props" Condition = " '$(FSharpTestCompilerVersion)' != '' "/>
-
-  <!-- 
+  <PropertyGroup>
+    <RepoRoot Condition="'$(RepoRoot)' == ''">$(MSBuildThisFileDirectory)</RepoRoot>
+  </PropertyGroup>
+  <!--
    When developers load the FSharp.Compiler.Service solution we set FSHARPCORE_USE_PACKAGE to true if it hasn't already been set to a value.
    This option ensures that building and testing uses the specified FSharp.Core nuget package instead of the local
    FSharp.Core project.
-   -->
-  <PropertyGroup Condition="'$(FSHARPCORE_USE_PACKAGE)' == ''">
-    <FSHARPCORE_USE_PACKAGE Condition="'$(SolutionName)' == 'FSharp.Compiler.Service'">true</FSHARPCORE_USE_PACKAGE>
+   We also disable arcade and reset certain artifacts and compiler paths to use default ones
+   All settings below can be overriden via CLI switches if needed. -->
+
+  <PropertyGroup Condition="'$(SolutionName)' == 'FSharp.Compiler.Service' and '$(BUILDING_USING_DOTNET)' != 'false'">
+      <BUILDING_USING_DOTNET Condition="'$(BUILDING_USING_DOTNET)' == ''">true</BUILDING_USING_DOTNET>
+  </PropertyGroup>
```

Comment:
Maybe it should be possible to just set `BUILDING_USING_DOTNET` when it's used in a different solution too?
We've been discussing trying to include FCS project in Rider and Fantomas solutions recently, and this check would be a problem there. cc @nojaf @safesparrow